### ISSUE TITLE
Segfault on skx when running gemm_ukr testsuite module

### ISSUE THREAD

**Issue by fgvanzee** [flame/blis#185]
I observe a segmentation fault, perhaps due to overwriting of internal data, when running the testsuite on Skylake server (skx). I configured as:
```
./configure --enable-cblas --enable-debug x86_64
```
though I'm not sure if any of this made a difference yet. (I assume that targeting `skx` instead results in the same error.)

Disabling the microkernel modules allows the testsuite to finish without error. Same result if I only request column storage testing via `input.general`. The first thing this makes me think of is that `skx` microkernels do not support output to row storage, even via a "general stride" branch. Care to comment, @devinamatthews, @dnparikh?
```
% --- gemm_ukr ---
% 
% test gemm_ukr seq front-end?    1
% gemm_ukr k                      -1
% gemm_ukr operand params         (none)
% 

% blis_<dt><op>_<stor>               m     n     k   gflops   resid      result

Program received signal SIGSEGV, Segmentation fault.
0x00002aaaab8e8418 in _int_free () from /lib64/libc.so.6
(gdb) bt
#0  0x00002aaaab8e8418 in _int_free () from /lib64/libc.so.6
#1  0x00000000005334ca in bli_free_align (f=0x2aaaab8ec4c0 <free>, p=0x1f410c0)
    at frame/base/bli_malloc.c:174
#2  0x00000000005332ff in bli_free_user (p=0x1f410c0) at frame/base/bli_malloc.c:64
#3  0x0000000000539a85 in bli_obj_free (obj=0x7fffffffa020) at frame/base/bli_obj.c:224
#4  0x0000000000428c8e in libblis_test_gemm_ukr_experiment (params=0x7fffffffb340, 
    op=0x7fffffffaf90, iface=BLIS_TEST_SEQ_UKERNEL, datatype=BLIS_FLOAT, 
    pc_str=0x1f32060 "", sc_str=0x1f31e90 "r", p_cur=100, perf=0x7fffffffa360, 
    resid=0x7fffffffa358) at src/test_gemm_ukr.c:312
#5  0x00000000004467d1 in libblis_test_op_driver (params=0x7fffffffb340, 
    op=0x7fffffffaf90, iface=BLIS_TEST_SEQ_UKERNEL, op_str=0x1831640 "gemm_ukr", 
    p_types=0x183164b "", o_types=0x1831649 "m", thresh=0x1f24620 <thresh>, 
    f_exp=0x428699 <libblis_test_gemm_ukr_experiment>) at src/test_libblis.c:1538
#6  0x000000000042868d in libblis_test_gemm_ukr (params=0x7fffffffb340, 
    op=0x7fffffffaf90) at src/test_gemm_ukr.c:132
#7  0x0000000000442055 in libblis_test_level3_ukrs (params=0x7fffffffb340, 
    ops=0x7fffffffa4c0) at src/test_libblis.c:175
#8  0x0000000000441bd9 in main (argc=5, argv=0x7fffffffb4a8) at src/test_libblis.c:91
```

EDIT: Can confirm that the bug manifests when simply configuring as `./configure skx`.

**Comment by devinamatthews** [comment]
They should. @dnparikh the sgemm kernels were all kinds of messed up at one point. Can you double-check what is currently in BLIS with the develop branch of TBLIS? @fgvanzee I suggest valgrind for further probing if that doesn't yield anything.

**Comment by dnparikh** [comment]
ca982148b3b419db063cad2fa74376ec383a5c80 should fix this issue.

**Comment by fgvanzee** [comment]
@devinamatthews @dnparikh Devangi has fixed this issue in ca98214, so I think we can close it now.
